backend/app.py

```diff
@@ -4,13 +4,20 @@
 import psycopg2
 from urllib.parse import unquote
 import os
+from flask_basicauth import BasicAuth
 from tasks.banner import BannerTask
 from tasks.username import UsernameTask
+import yaml
 
 app = Flask(__name__, static_folder='/backend/static', static_url_path='/')
 api = Api(app)
-password_decoded = unquote(os.environ.get('DB_PASSWORD'))
 
+with open(os.environ.get('CONFIG_PATH')) as f:
+    config = yaml.load(f, Loader=yaml.FullLoader)
+    app.config.update(config)
+basic_auth = BasicAuth(app)
+
+password_decoded = unquote(app.config.get('DB_PASSWORD'))
```

Comment:
if your `app.config` is something like this

```
{
  "basic_auth": {
    "username": "user",
    "password": "pass",
    "force": true
  },
  "db": {
    "host": "dbhost",
    "user": "dbuser",
    "password": "dbpassword",
    "name": "quay"
  }
}
```

Then your password is going to be `app.config.get('db', {}).get('password')`